lib/class-wp-duotone-gutenberg.php

```diff
@@ -349,7 +349,26 @@ public static function render_duotone_support( $block_content, $block ) {
 		$filter_id = gutenberg_get_duotone_filter_id( array( 'slug' => $slug ) );
 
 		// Build the CSS selectors to which the filter will be applied.
-		$selector = WP_Theme_JSON_Gutenberg::scope_selector( '.' . $filter_id, $duotone_selector );
+		$scopes    = explode( ',', '.' . $filter_id );
+		$selectors = explode( ',', $duotone_selector );
+
+		$selectors_scoped = array();
```

Comment:
I wish I hadn't to paste this here, but I need this to work differently than the `WP_Theme_JSON_Gutenberg::scope_selectors`.

Ideas for a follow-up PR:

- Create a utility `css_scope_selector( $scope, $selector ) ` function somewhere and make it work like this code (no spaces added).
- The consumer should be responsible for adding or not adding spaces:
  - `css_scope_selector( $scope . ' ', $selector )` if the consumer wants spaces.
  - `css_scope_selector( $scope . ' ', $selector )` if the consumer doesn't want spaces.

Thoughts?